packages/bot-web-ui/src/components/run-panel.jsx

```diff
@@ -122,42 +122,54 @@ const StatisticsInfoModal = ({ is_mobile, is_statistics_info_modal_open, toggleS
             <Modal.Body>
                 <div className={classNames('statistics__modal-body', { 'statistics__modal-body--mobile': is_mobile })}>
                     <ThemedScrollbars className='statistics__modal-scrollbar'>
-                        <p className='statistics__modal-body--content no-margin'>{localize('Total stake')}</p>
-                        <p>
+                        <Text as='p' weight='bold' className='statistics__modal-body--content no-margin'>
+                            {localize('Total stake')}
```

Comment:
just a question, not a suggestion 
As we use JSX, is it better if we have the functional or component type of localize ?

Reply:
They'll achieve the same result as the `Localize` component doesn't explicitly re-render when the language is changed. https://react.i18next.com/latest/trans-component

But if you can use a component, use that over an expression.